editor.planx.uk/src/@planx/components/List/utils.tsx

```diff
@@ -15,19 +24,27 @@ const List = styled("ul")(() => ({
  * @param field - the Field object
  * @returns string | React.JSX.Element - the `text` for the given value `val`, or the original value
  */
-export function formatSchemaDisplayValue(
-  value: string | string[],
-  field: Field,
-) {
+export const formatSchemaDisplayValue = <T extends Field>(
+  value: ResponseValue<T>,
+  field: T,
+) => {
   switch (field.type) {
-    case "number":
+    case "number": {
+      if (!isNumberFieldResponse(value)) return;
```

Comment:
These early `return`s are really just here for type narrowing and won't actually be hit- we know that a schema field will always match a response's field.